interfaces/index.ts

```diff
@@ -9,6 +9,16 @@ export interface Report {
     total_tax_deductions: number;
 }
 
+export interface ChatMessage {
+    id?: string | number;
```

Comment:
@yunjin08 I'd like to clarify something here, why can the id be either string or number? Should we decide on a single data type for this?